### PR TITLE
Bump to cf deployment 8.1

### PR DESCRIPTION
**Pull request by andreas-kupries**
## Description

See https://jira.suse.com/browse/CAP-460

Subordinate UAA PR = https://github.com/SUSE/uaa-fissile-release/pull/166

## Test plan

Build, start, pass all the test suites (:smoking: , brain, :cat:).

### CODE REVIEW

**Comment by jandubois** [comment]
Looks good, and I would have merged, except that the UAA submodule points to a commit on a feature branch, and there is no PR for merging that.

**Comment by andreas-kupries** [comment]
> Looks good, and I would have merged, except that the UAA submodule points to a commit on a feature branch, and there is no PR for merging that.

Given that the SCF PR had not passed Jenkins before I had not done the UAA PR yet. Now that this commit is good I will make the UAA PR ASAP.

**Comment by andreas-kupries** [comment]
And UAA PR is up.

**Comment by andreas-kupries** [comment]
Force pushed changes
- Removed removal of eirini properties. PEBKAC in interpreting the `diff releases` output. Ticket to to come to make that better.
- Integrated UAA bump.
- Bumped CATS to the version matching 9.5/deployment.

This should run through jenkins proper before getting merged.
A local vagrant run will be made as well.

**Comment by jandubois** [comment]
There is still another reference to `cflinuxfs2`:

```
make/tar-sources
5:DIRECTORY_BLACKLIST=".*-buildpack|cflinuxfs2|cflinuxfs3|sle12|sle15|opensuse42|opensuse15|busybox|golang.*"
```

File: README.md [hunk 1]
```diff
@@ -584,6 +584,7 @@ to remove the comments holding dump action tracing.
 1. Run `make diff-releases` to check the changed BOSH properties; see
    the next section for details.
 1. Run `tooling/bin/check-uaa-clients`
+1. Bump [CloudFoundry Acceptance Tests(CATs)](https://github.com/cloudfoundry/cf-acceptance-tests) submodule to match new `CF_VERSION`, before testing.
```
Comment (jandubois) [inline README.md]: I know you are just following the schema used above, but Markdown is supposed to be readable as plain text, and using the same number for each bullet point kind of defeats that purpose.

File: container-host-files/etc/scf/config/role-manifest.yml [hunk 2]
```diff
@@ -1408,12 +1412,31 @@ instance_groups:
...
+      properties.cc.ca_cert:                    ((INTERNAL_CA_CERT))
+      properties.cc.cert:                       ((RLP_GATEWAY_CERT))
+      properties.cc.common_name: api-group.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))
+      properties.cc.key:                        ((RLP_GATEWAY_CERT_KEY))
+      properties.http.cert:                     ((RLP_GATEWAY_CERT))
+      properties.http.key:                      ((RLP_GATEWAY_CERT_KEY))
+      properties.logs_provider.ca_cert:         ((INTERNAL_CA_CERT))
+      properties.logs_provider.client_cert:     ((SYSLOG_RLP_CERT))
+      properties.logs_provider.client_key:      ((SYSLOG_RLP_CERT_KEY))
+      #properties.logs_provider.common_name:    "reverselogproxy"
```
Comment (jandubois) [inline container-host-files/etc/scf/config/role-manifest.yml]: Why is this commented out? It should either be uncommented, or removed completely.

File: container-host-files/opt/scf/bin/klog.sh [hunk 3]
```diff
@@ -89,6 +89,8 @@ function get_all_events() {
 rm -rf "${KLOG:?}/${NS:?}"
 NAMESPACE_DIR="${KLOG}/${NS}"
 
+mkdir -p "${NAMESPACE_DIR}"
```
Comment (jandubois) [inline container-host-files/opt/scf/bin/klog.sh]: Why is this here? It seems unrelated to the topic of this PR.

File: README.md (hunk 1 above)
Comment (andreas-kupries) [reply README.md]: Sure, trivial to fix.

File: container-host-files/etc/scf/config/role-manifest.yml (hunk 2 above)
Comment (andreas-kupries) [reply container-host-files/etc/scf/config/role-manifest.yml]: Will remove (was simply a reminder of the default in the spec, and that this property need not be set)

File: container-host-files/opt/scf/bin/klog.sh (hunk 3 above)
Comment (andreas-kupries) [reply container-host-files/opt/scf/bin/klog.sh]: Will remove.

File: README.md (hunk 1 above)
Comment (andreas-kupries) [reply README.md]: Done with commit e7f3d54339f2021d8795054b2f97886a71f100b6

File: container-host-files/etc/scf/config/role-manifest.yml (hunk 2 above)
Comment (andreas-kupries) [reply container-host-files/etc/scf/config/role-manifest.yml]: Done with commit e7f3d54339f2021d8795054b2f97886a71f100b6

File: container-host-files/opt/scf/bin/klog.sh (hunk 3 above)
Comment (andreas-kupries) [reply container-host-files/opt/scf/bin/klog.sh]: Done with commit e7f3d54339f2021d8795054b2f97886a71f100b6

**Comment by andreas-kupries** [comment]
Done with commit e7f3d54339f2021d8795054b2f97886a71f100b6
